openqa-investigate

```diff
@@ -180,18 +180,60 @@ $comment_text"
     fi
 }
 
+fetch-investigate-results() {
+    local origin_job_id=$1 retry_id=$2
+    local state other_job_data state result investigate_type passed
+    local job="$(echo "$job_data" | runjq -r '.job')" || return $?
+
+    local IFS=$'\n'
+    local investigate_comment=($("${client_call[@]}" -X GET jobs/"$origin_job_id"/comments | runjq -r '[.[] | select(.text | contains("Automatic investigation jobs for job") and contains(":investigate:retry**:"))] | sort_by(.id) | first | .text')) || return $?
+
+    for line in "${investigate_comment[@]}"; do
+        if [[ $line =~ :investigate:([^*]+)\*\*:.*\/t([0-9]+) ]]; then
+            investigate_type=${BASH_REMATCH[1]}
+            other_id=${BASH_REMATCH[2]}
+            if [[ $other_id -ne $retry_id ]]; then
```

Comment:
```suggestion
            if [[ $other_id -ne "$retry_id" ]]; then
```

Reply:
Hm, shellcheck doesn't complain here, but I changed it